README.md

```diff
@@ -115,6 +115,8 @@ docker exec spark-node-master ./submit.sh --json-file /tmp/jobs.json
 
 The command line interface also has multiple optional flags:
 
+- `--extract-target` - iasWorld target environment to extract data from. Must
```

Comment:
I don't expect that we'll need to pull data from any further environments (unless there are some I don't know about), so keeping this limited to two choices.